stacks-signer/src/config.rs

```diff
@@ -36,7 +36,7 @@ use wsts::state_machine::PublicKeys;
 /// List of key_ids for each signer_id
 pub type SignerKeyIds = HashMap<u32, Vec<u32>>;
 
-const EVENT_TIMEOUT_MS: u64 = 5000;
+const EVENT_TIMEOUT_MS: u64 = 50;
```

Comment:
That's a little small, don't you think?

Reply:
The issue here is that the libsigner runloop has to multiplex two event receivers: one with stackerdb events, the other with main loop commands.  So every time we would get an event without a command, we'd wait 5 seconds for a command that never came.

I'm still not exactly sure *why* this was suddenly a problem.  `wsts v7` added a `DkgEndBegin` message, which is being amplified by all of the inactive coordinators, so presumably that was it.  But I have no idea why we weren't often hitting this issue, since we don't send many commands but received lots of messages even before this change.

Regardless, this was always broken before, so the fix seems appropriate.

Reply:
I think once we remove the command receiver, we can update it back to 5000 ms. However, for now I think this fix is okay. If we find we need to KEEP the command receiver we should have a sep timeout for it :)